apps/picsa-tools/climate-tool/src/app/pages/site-select/site-select.page.ts

```diff
@@ -57,6 +57,31 @@ export class SiteSelectPage {
     });
   }
 
+  ngOnInit() {
+    this.route.queryParams.subscribe((params) => {
+      if (!params['forceSelect']) {
```

Comment:
Ah, interesting challenge! I can see that you are trying to enable the user to come back to the site select page even when a site has been selected. I think an easier solution would be to just remove the site selected - which you tried to do, but there was a bug so probably why you added this after. 

Not a bad idea at all, but would recommend trying to get the simpler solution to work first before adding a more complicated one